app/components/build/AnnotationsList.js

```diff
@@ -162,7 +212,13 @@ export default Relay.createContainer(AnnnotationsList, {
                 html
               }
             }
+            cursor
+          }
+          pageInfo {
+            hasNextPage
+            hasPreviousPage
```

Comment:
If we rely on `ShowMoreFooter` to handle the paging info, we don’t need to pull in either of these into this component!

Reply:
That's very cool, thanks for pointing it out :)